spine_json_lib/test/test_utils.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+from spine_json_lib.utils import get_tags_from_name
+from deepdiff import DeepDiff
+
+
+@pytest.mark.parametrize(
+    "tag_name, result",
+    [
+        ("master[scale:0.5]", {"scale": 0.5}),
+        ("master[ scale:0.5 ]", {"scale": 0.5}),
+        ("master[scale:0.5]", {"scale": 0.5}),
```

Comment:
I think first parametrize and third one are the same, aren't they? maybe you want to add parametrize like`master[scale : 0.5]` or `master[ scale : 0.5 ]`

Reply:
Good catch, blame `black` who changed the code removing some of the spaces 😄